### PR TITLE
fix: make edits to Contributing guide

### PR DESCRIPTION
**Pull request by CBID2**
Closes #issue 
N/A

> [!NOTE]

> This is an entry for the bug challenge

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ Why should you contribute to Codédex? Contributing will get you:
 2. Once your project topic is accepted by the Codédex team, you can now start writing it!
 3. Make a copy of the [project template](https://github.com/codedex-io/projects/blob/main/docs/project_template.mdx), then review our [content guidelines](https://github.com/codedex-io/projects/tree/main/docs/content_guidelines.mdx), the [official MDX docs](https://mdxjs.com/docs/), and 
 [one of our published projects](https://github.com/codedex-io/projects/blob/main/projects/generate-a-qr-code-with-python/generate-a-qr-code-with-python.mdx) 
-to see what a final project should look like. To make sure your work renders properly, check it with an MDX previewer like [this one](https://mdxjs.com/playground/).
+To see what a final project should looks like and ensure that it renders properly, use an MDX previewer like [this one](https://mdxjs.com/playground/).
```
Comment (intelagense) [inline .github/CONTRIBUTING.md:23]: This line still seems a bit off. I thinks `looks` should remain as `look`

**Review by intelagense — CHANGES_REQUESTED** [review]
Looks great overall!

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (CBID2) [inline .github/CONTRIBUTING.md:23]:
```suggestion
To see what a final project should look like and ensure that it renders properly, use an MDX previewer like [this one](https://mdxjs.com/playground/).
```

**Review by CBID2 — COMMENTED** [review]
Turn back a word into look

**Comment by CBID2** [comment]
> Looks great overall!

Done @intelagense

**Comment by intelagense** [comment]
Thanks!

**Pull request merged**